index.bs

```diff
@@ -1283,22 +1283,22 @@ defined in [[!WebCryptoAPI]].
 The API defined in this specification implies a specific abstract functional model for an [=authenticator=]. This section
 describes the authenticator model.
 
-Client platforms may implement and expose this abstract model in any way desired. However, the behavior of the client's Web
+Client platforms MAY implement and expose this abstract model in any way desired. However, the behavior of the client's Web
```

Comment:
This is descriptive - not normative language. Please back this individual change out.

Reply:
done.